src/passkit.cs

```diff
@@ -1316,4 +1434,386 @@ interface PKSecureElementPass {
+
+		[Export ("passThumbnailImage")]
+		CGImage PassThumbnailImage { get; }
+
+		[Export ("localizedDescription", ArgumentSemantic.Strong)]
+		string LocalizedDescription { get; }
+
+		[Export ("ownerDisplayName", ArgumentSemantic.Strong)]
+		string OwnerDisplayName { get; }
+	}
+
+	[NoWatch, NoTV]
+	[iOS (14,0)]
+	[Mac (11,0)]
+	[BaseType (typeof (PKAddSecureElementPassConfiguration))]
+	[DisableDefaultCtor]
+	interface PKAddShareablePassConfiguration {
+
+		[Async]
+		[Static]
+		[Export ("configurationForPassMetadata:provisioningPolicyIdentifier:primaryAction:completion:")]
+		void GetConfiguration (PKShareablePassMetadata[] passMetadata, string provisioningPolicyIdentifier, PKAddShareablePassConfigurationPrimaryAction action, Action<PKAddShareablePassConfiguration, NSError> completion);
+
+		[Export ("primaryAction")]
+		PKAddShareablePassConfigurationPrimaryAction PrimaryAction { get; }
+
+		[Export ("credentialsMetadata", ArgumentSemantic.Strong)]
+		PKShareablePassMetadata[] CredentialsMetadata { get; }
+
+		[Export ("provisioningPolicyIdentifier", ArgumentSemantic.Strong)]
+		string ProvisioningPolicyIdentifier { get; }
+	}
+
+	[Mac (11,0)]
+	[Watch (7,0)][iOS (14,0)]
+	[NoTV]
+	[BaseType (typeof (NSObject))]
+	[DisableDefaultCtor]
+	interface PKBarcodeEventConfigurationRequest {
+
+		[Export ("deviceAccountIdentifier")]
+		string DeviceAccountIdentifier { get; }
+
+		[Export ("configurationData")]
+		NSData ConfigurationData { get; }
+
+		[Export ("configurationDataType")]
+		PKBarcodeEventConfigurationDataType ConfigurationDataType { get; }
+	}
+
+	[Mac (11,0)]
+	[Watch (7,0)][iOS (14,0)]
+	[NoTV]
+	[BaseType (typeof (NSObject))]
+	[DisableDefaultCtor]
+	interface PKBarcodeEventMetadataRequest {
+
+		[Export ("deviceAccountIdentifier")]
+		string DeviceAccountIdentifier { get; }
+
+		[Export ("lastUsedBarcodeIdentifier")]
+		string LastUsedBarcodeIdentifier { get; }
+	}
+
+	[Mac (11,0)]
+	[Watch (7,0)][iOS (14,0)]
+	[NoTV]
+	[BaseType (typeof (NSObject))]
+	[DisableDefaultCtor]
+	interface PKBarcodeEventMetadataResponse {
+
+		[Export ("initWithPaymentInformation:")]
+		IntPtr Constructor (NSData paymentInformation);
+
+		[Export ("paymentInformation", ArgumentSemantic.Copy)]
+		NSData PaymentInformation { get; set; }
+	}
+
+	[Mac (11,0)]
+	[Watch (7,0)][iOS (14,0)]
+	[NoTV]
+	[BaseType (typeof (NSObject))]
+	[DisableDefaultCtor]
+	interface PKBarcodeEventSignatureRequest {
+
+		[Export ("deviceAccountIdentifier")]
+		string DeviceAccountIdentifier { get; }
+
+		[Export ("transactionIdentifier")]
+		string TransactionIdentifier { get; }
+
+		[Export ("barcodeIdentifier")]
+		string BarcodeIdentifier { get; }
+
+		[Export ("rawMerchantName")]
+		string RawMerchantName { get; }
+
+		[Export ("merchantName")]
+		string MerchantName { get; }
+
+		[Export ("transactionDate", ArgumentSemantic.Strong)]
+		NSDate TransactionDate { get; }
+
+		[Export ("currencyCode")]
+		string CurrencyCode { get; }
+
+		// FIXME ?
```

Comment:
? Should we create an issue? or list it in our bucket for review issue

Reply:
I forgot to remove the `FIXME` 
Other places (where `amount` is defined, in PassKit and elsewhere) uses `NSDecimalNumber` which is a subclass of `NSNumber` so we can't bind it as such.